CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## [Unreleased]
 
+## [0.0.3] - 2024-09-05
```

Comment:
In general, you wouldn't know which release this would be part of. That's why you should put it under unreleased. The person responsible for making a new release should make appropriate changes to the changelog